Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceStateMachine+Action.swift

```diff
@@ -10,12 +10,13 @@ import Foundation
 
 @available(iOS 13.0, *)
 extension ExperienceStateMachine {
-    enum Action {
+    indirect enum Action {
```

Comment:
`indirect` since new `reportError` can contain a `SideEffect`, and a `SideEffect` can contain an `Action` (i.e. `.continuation(Action)`)